src/ScreenMapControllers.cpp

```diff
@@ -278,10 +278,14 @@ void ScreenMapControllers::Update( float fDeltaTime )
 		}
 	}
 
-	//
-	// Update devices text
-	//
-	m_textDevices.SetText( INPUTMAN->GetDisplayDevicesString() );
+	// Update devices text every 500 frames.
+	// This is NOT updating the actual text shown!
```

Comment:
I am confused as to how it's not updating the actual text. SetText... sets the text?

Reply:
I would say something like:

// We don't expect the connected devices to change frequently.
// As a result, delay how often we check for and update the device strings.